closure/goog/editor/plugins/removeformatting.js

```diff
@@ -651,8 +651,21 @@ goog.editor.plugins.RemoveFormatting.prototype.removeFormattingWorker_ =
           continue;
 
         case goog.dom.TagName.P:
-          goog.editor.plugins.RemoveFormatting.appendNewline_(sb);
-          goog.editor.plugins.RemoveFormatting.appendNewline_(sb);
+
+		  // Make sure the "P" paragraph tag is not the first
```

Comment:
I apologize for this - it is not. I logged a number of separate pull requests and the change to this file got caught in this PR - PR #472.

For PR #472, only file

closure/goog/demos/keyhandler.html

should change.